src/util/types.h

```diff
@@ -46,6 +46,71 @@ std::string formatSize(size_t size);
 // returns true if the asset is well formed for the specified protocol version
 template <typename T> bool isAssetValid(T const& cur, uint32_t ledgerVersion);
 
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+uint32_t getExpiration(LedgerEntry const& e);
```

Comment:
nit: I'm not sure this is the right place for `LedgerEntry` utils - somewhere in `ledger` folder would be more appropriate.